frontend/src/components/PhotoUpload.css

```
@@ -240,3 +240,78 @@
    object-fit: cover;
    border-radius: 14px;
 }
+
+/* Menu item search dropdown styles */
+.menu-item-search-input {
+   width: 100%;
+   height: 52px;
+   padding: 0 14px;
+   border-radius: 20px;
+   border: 2px solid #ff6a39;
+   font-family: "Poppins", sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)